Sources/NIOPosix/Windows.swift

```diff
@@ -59,6 +59,54 @@ extension NIOCore.Windows {
         }
         return .processed(Int(result))
     }
+
+    static func pread(
+        descriptor: CInt,
+        pointer: UnsafeMutableRawPointer,
+        size: size_t,
+        offset: off_t
+    ) throws -> IOResult<ssize_t> {
+        var overlapped = OVERLAPPED()
+        // off_t is Int32 anyway. Therefore high is always zero.
```

Comment:
Can you add a precondition to that effect? It'll help us be confident.

Reply:
fixed.